src/store.js

```diff
@@ -559,7 +559,7 @@ function EvolvStore(options) {
         return;
       }
 
-      let configLoaded = true;
+      let configLoaded = !!config;
```

Comment:
Thanks @mattstrom , this is the permanent fix? Can you talk about what's going on here?

Reply:
@teolisitza Yes, this would be the permanent fix.

You'll see a few lines down from this line that the `update()` call is short-circuited when `configLoaded` is not true. However, what was happening previously was that `configLoaded` was true by default; because requests were synchronous, it was an accurate reflection of the state of the asset manager. But now that requests are asynchronous, we need to actually check if `config` has been loaded with a value.